### ISSUE TITLE
Organización de archivos

### ISSUE THREAD

**Issue by LuluAylen00** [0623CIFSNCN05LAED/grupo-07#29]
Tengan cuidado, su proyecto posee una gran cantidad de archivos residuales que no poseen ninguna funcionalidad en este momento
Hay 7 archivos en la carpeta de rutas de los cuales solo usan 1 (y el resto no está conectado de ninguna manera a este)
De la misma manera, hay 7 archivos de controladores de los cuales también usan uno único.
Lo óptimo sería la división de archivos dependiendo del área que se encargan (con un archivo principal, uno de productos y uno de usuarios podría alcanzar, por ejemplo).
En el peor de los casos, se requiere la eliminación de los archivos en desuso para evitar tenerlos de manera residual en el proyecto

**Comment by JonathanUGomez93** [comment]
Sí, Aylu! Vamos a eliminarlos. Pasa que estábamos terminando de trabajar en ellos y, una vez completo todo lo que querías hacer, vamos a eliminarlos y a su vez separar el archivo único en sus respectivas versiones de general, producto y usuario.
